MuMuPiKPAT/test/sanjay/selector/psiPrimePiK_MC.C

```diff
@@ -3044,11 +3061,25 @@ Bool_t psiPrimePiK_MC::Process(Long64_t entry)
 		      hmyPsiPKPiMassCutsSelAlt->Fill( B0Mass_1B0_hardCuts );
 		      hmyPsiPPiPiMassCutsSelAlt->Fill( (PsiPp4_orig + PiPip4_orig).M() );
 
-		      if ( fabs(myB0MassAlt - B0_massFit) < B0_signal*B0_sigmaFit )
+		      if ( fabs(myB0MassAlt - B0_massFit) < B0_signal*B0_sigmaFit ) {
 			psi2SPiMassSq_vs_KPiMassSq_CutsPeak->Fill( KPip4_orig.M2(), Zp4_orig.M2() ) ;
+                        hardcuts_vars_sig[0] = KPi_hC;
+                        hardcuts_vars_sig[1] = psiPi_hC;
+                        hardcuts_vars_sig[2] = cosTheta_psi_hC;
+                        hardcuts_vars_sig[3] = phi_hC;
+                        hardcuts_vars_sig[4] = B0beauty_hC;
+                        hardcuts_variables_sig->Fill();
+                      }
 		      else
-			if ( (fabs(myB0MassAlt - B0_massFit) > innerSB*B0_sigmaFit)  &&  (fabs(myB0MassAlt - B0_massFit) < outerSB*B0_sigmaFit) )
+			if ( (fabs(myB0MassAlt - B0_massFit) > innerSB*B0_sigmaFit)  &&  (fabs(myB0MassAlt - B0_massFit) < outerSB*B0_sigmaFit) ) {
 			  psi2SPiMassSq_vs_KPiMassSq_CutsSb->Fill( KPip4_orig.M2(), Zp4_orig.M2() ) ;
+                          hardcuts_vars_bkg[0] = KPi_hC;
+                          hardcuts_vars_bkg[1] = psiPi_hC;
+                          hardcuts_vars_bkg[2] = cosTheta_psi_hC;
+                          hardcuts_vars_bkg[3] = phi_hC;
+                          hardcuts_vars_bkg[4] = B0beauty_hC;
+                          hardcuts_variables_bkg->Fill();
```

Comment:
Same as in
https://github.com/lecriste/Z4430/pull/1/files#diff-2aef6870c472c96f6a5aec4107f14debR4234